demo/package.json

```diff
@@ -12,23 +12,23 @@
     "test:mock": "nodemon test/nodeMock/app.js --standalone"
   },
   "dependencies": {
-    "@angular-devkit/build-angular": "0.10.5",
+    "@angular-devkit/build-angular": "~0.13.0",
     "@angular-material-components/datetime-picker": "^2.0.4",
     "@angular-material-components/moment-adapter": "^5.0.0",
-    "@angular/animations": "~7.0.3",
-    "@angular/cdk": "^7.0.3",
-    "@angular/cli": "^7.0.3",
-    "@angular/common": "^7.0.3",
-    "@angular/compiler": "7.0.3",
-    "@angular/compiler-cli": "7.0.3",
-    "@angular/core": "^7.0.3",
-    "@angular/forms": "~7.0.3",
-    "@angular/http": "~7.0.3",
-    "@angular/material": "^7.0.3",
-    "@angular/platform-browser": "^7.0.3",
-    "@angular/platform-browser-dynamic": "^7.0.3",
-    "@angular/router": "~7.0.3",
-    "@angular/upgrade": "~7.0.3",
+    "@angular/animations": "^7.2.13",
+    "@angular/cdk": "^7.3.7",
+    "@angular/cli": "^6.0.8",
```

Comment:
@johnbenjamin-hmcts - quick query on the apparent downgrade of `@angular/cli` version... is this intentional?

Reply:
Hi @Daniel-Lam, I tried to keep CCD UI Toolkit in sync with MC, and in MC the version of `@angular/cli` is 6.0.8

Reply:
@johnbenjamin-hmcts - oh, OK... fair enough. At least it's consistent with MC :)